tests/bluetooth/tester/src/btp/btp_aics.h

```diff
@@ -0,0 +1,26 @@
+/* aics.h - Bluetooth tester headers */
+
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
+#include <zephyr/sys/util.h>
+
+#define AICS_READ_SUPPORTED_COMMANDS	0x01
```

Comment:
Should we be prefixing these with `BTP_`?

Reply:
We spoke about in PR that aimed for splitting btp, and the final decision was to leave those defines without the BTP_ prefix.

Reply:
The discussion you refer to is https://github.com/zephyrproject-rtos/zephyr/pull/52246#discussion_r1026173386 right? 

It was not decided to keep as BTP, but rather the consensus was that it would be a good idea to modify, but should be done in AutoPTS as well. IMO it would be better to add `BTP` here and open a PR for AutoPTS as well, rather than continuing doing it differently than what we want. Unfortunately I never got to re-review https://github.com/zephyrproject-rtos/zephyr/pull/52246 before it got merged :) 

I will still argue that we should add `BTP_` here, and open the AutoPTS PR ASAP, but I won't block this PR based on this as it can be done later

Reply:
I would skip the `BTP_` prefix for now to keep it consistent with other definitions e.g https://github.com/zephyrproject-rtos/zephyr/blob/main/tests/bluetooth/tester/src/btp/btp_l2cap.h

Reply:
Fair :)